Hamzah/library_challenge.py

```diff
@@ -0,0 +1,196 @@
+print('Question 1:')
+# You are working on a library management system, here are the list books at the library
+books = ['MY OWN WORDS', 'WHITE FRAGILITY', 'THE BODY KEEPS THE SCORE', 'SO YOU WANT TO TALK ABOUT RACE', 'STAMPED FROM THE BEGINNING', 'JUST MERCY', 'BORN A CRIME',
+         'THE WARMTH OF OTHER SUNS', 'THE COLOR OF LAW', 'THE NEW JIM CROW', 'THE TRUTHS WE HOLD', 'SAPIENS', 'BRAIDING SWEETGRASS', "MY GRANDMOTHER'S HANDS", 'ON TYRANNY']
+
+# 1.0
+# What data type is the object 'books'? How do you know?
+
+print(type(books))
+print('The data type is a list of string because of the bracket ([]) and strings inside')
```

Comment:
this only required a comment; however, I'm glad you used proper syntax to answer the question within a string